nautobot_ssot/models.py

```diff
@@ -47,6 +47,11 @@ class Sync(BaseModel):
 
     start_time = models.DateTimeField(blank=True, null=True)
     # end_time is represented by the job_result.completed field
+    load_time_source = models.DurationField(blank=True, null=True)
+    load_time_target = models.DurationField(blank=True, null=True)
+    diff_time = models.DurationField(blank=True, null=True)
+    sync_time = models.DurationField(blank=True, null=True)
+    memory_usage = models.IntegerField(blank=True, null=True)
```

Comment:
Planning to add these to the sync detail page template? In general what's your intended purpose of storing these in the DB given they're already also present in the job logs?

Reply:
the idea is following @dgarros 's point about storing them for future retrieval from external source, not only as text.
The idea would to present them as metrics, or directly in the UI/API (not done yet, to confirm).